### ISSUE TITLE
Code block on methodology page lost syntax highlighting

### ISSUE THREAD

**Issue by rviscomi** [HTTPArchive/almanac.httparchive.org#556]
On https://almanac.httparchive.org/en/2019/methodology#dataset

![image](https://user-images.githubusercontent.com/1120896/69724838-b8615680-10ea-11ea-9492-5c9b102353d4.png)

I've manually applied syntax highlighting spans and classes to the query but the style selectors are not matching anymore. I think a parent class was renamed or the stylesheet moved or something.

**Comment by catalinred** [comment]
I dug a bit into this because at first, I thought I messed this up with one of my old commits. 😱 

Even though I modified `methodology.html` and `methodology.css` in the past, the latter I only modified it for [merging CSS duplicated rules](https://github.com/HTTPArchive/almanac.httparchive.org/commit/71e0e3e1afbd629f2153da9d098f5eb15047f9a3#diff-99747f4a7bab79e7de9502f180a97932). Then I remembered [I also did notice](https://github.com/HTTPArchive/almanac.httparchive.org/pull/471#issuecomment-553340163) that `methodology.css` wasn't used/loaded anymore in any page.

So I went further and I played Sherlock a bit. 🕵 
I found that the `methodology.css` was added in `methodology.html` by @rviscomi on [Oct 18](https://github.com/HTTPArchive/almanac.httparchive.org/commit/6bb9efa64cd938d6fe7d56550d09a12b9d471401#diff-99747f4a7bab79e7de9502f180a97932) and seem to have been removed by @tyohan on [Oct 29](https://github.com/HTTPArchive/almanac.httparchive.org/commit/77946a9f82d08e689f65dd02f2e42c1d415efced#diff-99747f4a7bab79e7de9502f180a97932).

Anyways, I re-added the missing `.code-block` bits to currently in use `page.css` 
Fixed in #573
